### DISCUSSION TITLE
Vehicle-in-Vehicle Crate Loading

### DISCUSSION THREAD

**Discussion by SgtKrunch** [Vdauphin/HeartsAndMinds#1162] [Question and answer]
Does this work with spawned crates as well? Haven't had a chance to test it, but it would be really cool and I see that there is some ViV scripts going on. If you can do it with crates, what is the procedure to do it? I apologize if this is a dumb question.

**Comment by Vdauphin** [comment]
Hello,
It is not easy to do it directly in mission but I opened a pull request on ACE repo as the integration of such feature will be easier: https://github.com/acemod/ACE3/pull/7984
Cheers

**Reply by SgtKrunch** [reply]
Awesome
